pkg/nvml/api.go

```diff
@@ -349,6 +349,210 @@ type Interface interface {
 	GetVgpuDriverCapabilities(Capability VgpuDriverCapability) (bool, Return)
 }
 
+type Device interface {
+	GetName() (string, Return)
```

Comment:
Can we sort these? (even if it is as a follow-up?)

Reply:
OK. Ordering the same as `nvml.h` is good -- this should then aid automation and also make diffs easy enough to read. No need for this follow up then.